conan/tools/cmake/cmakedeps/templates/target_configuration.py

```diff
@@ -14,8 +13,9 @@ class TargetConfigurationTemplate(CMakeDepsFileTemplate):
 
     @property
     def filename(self):
-        return "{}Target-{}.cmake".format(self.file_name,
-                                          self.cmakedeps.configuration.lower())
+        name = "" if not self.find_module_mode else "module-"
```

Comment:
I was not aware about this "module" prefix, I thought the .cmake scripts had to be called ``find_zlib.cmake`` or something similar.

Reply:
The `FindZLib.cmake` file is not prefixed with `module`, only the other files that are included from it.

Reply:
Is there a specific example where this is being done? look at https://gitlab.kitware.com/cmake/cmake/-/tree/master/Modules I did not find any and I have yet to compress this before